generator.js

```diff
@@ -189,6 +189,7 @@ module.exports = class WebpackGenerator extends Generator {
 
 	installPlugins() {
 		this.runInstall(getPackageManager(), this.dependencies, {
+			"dev": true,
```

Comment:
I don't think it is correct way to do this. Can we check the package manager?

Reply:
Working on it @sendilkumarn.